app/assets/stylesheets/v3/layout_components/buttons/button-grouping.scss

```diff
@@ -137,8 +150,17 @@
   }
 
   &__text {
-    line-height: 1rem;
+    line-height: 1.5;
```

Comment:
this should have a unit?

Reply:
Does it need it?

This is what I found about this subject:

"Unitless line heights are recommended due to the fact that child elements will inherit the raw number value, rather than the computed value. With this, child elements can compute their line heights based on their computed font size, rather than inheriting an arbitrary value from a parent that is more likely to need overriding."